src/trove_classifiers/__init__.py

```diff
@@ -890,3 +890,10 @@
 all_classifiers: List[str] = sorted(
     sorted_classifiers + list(deprecated_classifiers.keys())
 )
+
+__all__ = [
+    "all_classifiers",
+    "deprecated_classifiers",
+    "sorted_classifiers",
+    "sorted_classifiers",
```

Comment:
Is there a reason `sorted_classifiers` is here twice? I think you mean:

```suggestion
    "all_classifiers",
    "classifiers",
    "deprecated_classifiers",
    "sorted_classifiers",
```